### ISSUE TITLE
Overriding command help formatting?

### ISSUE THREAD

**Issue by alexagranov** [slack-ruby/slack-ruby-bot#172]
Hi @dblock - 

So I'm finding that our team would like to deviate from how command help is currently formatted.  Yet I would still like to employ the existing attribute helpers for specifying command `desc` and `long_desc`.   How about something like this instead of https://github.com/slack-ruby/slack-ruby-bot/blob/master/lib/slack-ruby-bot/support/commands_helper.rb#L35:

`block_given? ? yield : "#{command_name_and_desc(help_attrs)}\n\n#{help_attrs.command_long_desc}"`

And then usage like:
```
commands_helper = SlackRubyBot::CommandsHelper.instance
command_help = commands_helper.command_full_desc(opts[:command]) do
  help_attrs = commands_helper.send(:find_command_help_attrs, opts[:command])
  "#{help_attrs.command_desc}\n\n#{help_attrs.command_long_desc}"
end
```

What do you think?

**Comment by dblock** [comment]
You can just override that method, no?

**Comment by alexagranov** [comment]
Not really, because `help_attrs` isn't exposed.  So I'd end up having to call `find_command_help_attrs` myself as well (after having delegated to `super` to pick up the nice validation/error-handling logic).

**Comment by dblock** [comment]
Always prefer basic inheritance first, so feel free to refactor this/these classes. I think a block here feels like a weird interface.

**Comment by alexagranov** [comment]
It does feel weird.  Maybe the key is to just expose `help_attrs` and then we can fall back to basic inheritance?

**Comment by dblock** [comment]
I would prefer that.

**Comment by dblock** [comment]
@alexagranov Should this be closed with #173?

**Comment by alexagranov** [comment]
@dblock Sure looks like PR #182 may have done the trick.  Thanks for following up!
